mmv1/third_party/terraform/services/logging/resource_logging_project_sink.go

```diff
@@ -33,13 +33,6 @@ func ResourceLoggingProjectSink() *schema.Resource {
 		ForceNew:    true,
 		Description: `The ID of the project to create the sink in. If omitted, the project associated with the provider is used.`,
 	}
```

Comment:
```suggestion
	}
	schm.Schema["unique_writer_identity"] = &schema.Schema{
		Type:        schema.TypeBool,
		Optional:    true,
		Default:     false,
		Description: `Whether or not to create a unique identity associated with this sink. If false (the default), then the writer_identity used is serviceAccount:cloud-logs@system.gserviceaccount.com. If true, then a unique service account is created and used for this sink. If you wish to publish logs across projects, you must set unique_writer_identity to true.`,
	}
```

This is likely the change you are after.  Removing the whole field from the schema will make terraform completely unaware of the field in general